unit_test/build_ids_test.py

```diff
@@ -1,19 +1,11 @@
+import pytest
 import toml
-from toml.encoder import TomlEncoder
 
 from cibuildwheel.util import resources_dir
 
+Version = pytest.importorskip("packaging.version").Version
```

Comment:
We don't (yet) require this to install, so it might be missing - this test is skipped in that case. Now with the CIBW_REQUIRES_PYTHON addition, we would require packaging, so this would go away.